tests-invoke

```diff
@@ -96,7 +96,7 @@ def main():
                 time.sleep(int(os.getenv("TEST_INVOKE_SLEEP", "60")))
         return_code = p.returncode
         sys.stderr.write(f"Test run finished, return code: {return_code}\n")
-        if not api.has_open_prs(opts.revision) and not collision_detected and return_code != 0:
+        if not opts.pull_number and not collision_detected and return_code != 0:
```

Comment:
wasn't `pull_number` 0 for testing?

Reply:
It is, but for nightly we don't pass it to tests-invoke:

```
../tests-invoke --revision d8c2edc8a2e611f4b65285434bad0f08ebff140a --repo cockpit-project/cockpit'
```

See: https://cockpit-logs.us-east-1.linodeobjects.com/pull-0-20240222-012334-d8c2edc8-fedora-39-daily/log

Reply:
I'm not 100% sure, but let's just robustify this. Updated.